.vscode/settings.json

```diff
@@ -0,0 +1,47 @@
+// Recall to run the following command before even run vscode.
```

Comment:
Does this work? JSON doesn't have comments syntax - so this file is not valid JSON. (It's possible that the vscode JSON parser allows this, but good to check).

Reply:
Looked it up - vscode configuration files use "JSON with comments" so this is fine. https://code.visualstudio.com/docs/languages/json#_json-with-comments